com.unity.ml-agents/Runtime/Sensors/BufferSensor.cs

```diff
@@ -0,0 +1,95 @@
+using System;
+
+namespace Unity.MLAgents.Sensors
+{
+    public class BufferSensor : ISensor, IDimensionPropertiesSensor
```

Comment:
(probably my own ignorance but) why do the sensors implement both interfaces when IDimensionPropertiesSensor inherits from ISensor?

Reply:
The original proposal was that IDimensionPropertiesSensor would be its own interface and not inherit from ISensor.

Reply:
Ah I see, it has changed 👍